Doc/library/cgitb.rst

```diff
@@ -42,16 +42,25 @@ analysis.
    .. index:: single: excepthook() (in module sys)
 
    This function causes the :mod:`cgitb` module to take over the interpreter's
-   default handling for exceptions by setting the value of :attr:`sys.excepthook`.
-
-   The optional argument *display* defaults to ``1`` and can be set to ``0`` to
-   suppress sending the traceback to the browser. If the argument *logdir* is
-   present, the traceback reports are written to files.  The value of *logdir*
-   should be a directory where these files will be placed. The optional argument
-   *context* is the number of lines of context to display around the current line
-   of source code in the traceback; this defaults to ``5``. If the optional
-   argument *format* is ``"html"``, the output is formatted as HTML.  Any other
-   value forces plain text output.  The default value is ``"html"``.
+   default handling for exceptions by setting the value of
+   :attr:`sys.excepthook`.
+
+   The optional argument *display* defaults to ``1`` and can be set to ``0``
+   to suppress sending the traceback to the browser. A message single line
```

Comment:
Add another blank space after the full stop.

Reply:
Existing text does not seem to use that convention in this file!